### DISCUSSION TITLE
Maintain bot orders between campaign missions

### DISCUSSION THREAD

**Discussion by Rushyo** [FakeFishGames/Barotrauma#5578] [Ideas and suggestions]
It would be a big QoL improvement to not have to give bots new orders at the start of every mission of the campaign when they're unlikely to change between missions, particularly later on in a single-player campaign when there are more bots to manage.

**Comment by ekqvist** [comment]
Hi! This is something that we are currently working on internally so it's definitely something we would like to make happen.

**Comment by ekqvist** [comment]
I'm happy to say that this has now been implemented in the Unstable version!

**Comment by CurePindal** [comment]
Why is this still here? Its already implemented!
